openprescribing/dmd/models.py

```diff
@@ -183,3 +187,28 @@ def unreconciled(self):
             return self.filter(vmpp__isnull=True)
 
     objects = Manager()
+
+
+class TariffPrice(models.Model):
+    """Price
+    """
+    date = models.DateField(db_index=True)
+    vmpp = models.ForeignKey(DMDVmpp)
+    product = models.ForeignKey(DMDProduct)
+    # 1: Category A
+    # 3: Category C
+    # 11: Category M
+    tariff_category = models.ForeignKey(TariffCategory)
+    price_pence = models.IntegerField()
+
+    @property
+    def concession(self):
```

Comment:
We can't have a foreign key here, but could you change the property to a method called `get_concession`?